src/settings/Settings.ts

```diff
@@ -607,4 +607,8 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("Enable experimental, compact IRC style layout"),
         default: false,
     },
+    "Widgets.pinned": {
+        supportedLevels: LEVELS_ROOM_OR_ACCOUNT,
+        default: {},
+    },
```

Comment:
I'm not super convinced these sorts of things need to be settings if they don't get names and can't describe their types. At the very least, this needs to describe what the object is.

See also: https://github.com/vector-im/element-web/issues/8894

Reply:
Sure but then having to handle local echo on room-account-data separately feels a pain

Reply:
bleh, true. Its probably fine for now and we can add it to 8894 to validate